implementations/rust/ockam/ockam_abac/src/attribute_access_control.rs

```diff
@@ -54,21 +57,36 @@ impl AbacAccessControl {
     /// a message has an authenticated attribute with the correct name and value
     pub fn create(
         identity_attributes_repository: Arc<dyn IdentityAttributesRepository>,
+        authority: Identifier,
         attribute_name: &str,
         attribute_value: &str,
-    ) -> AbacAccessControl
-where {
+    ) -> AbacAccessControl {
         let expression = List(vec![
             Ident("=".into()),
             Ident(format!("subject.{attribute_name}")),
             Str(attribute_value.into()),
         ]);
         AbacAccessControl::new(
             identity_attributes_repository,
+            authority,
             Policy::new(expression),
             Env::new(),
         )
     }
+
+    /// Create an AccessControl which will verify that the sender of
+    /// a message has an authenticated credential without checking any attributes
+    pub fn create_empty(
```

Comment:
I'm not sure if `empty` really carries the meaning set in the comment. What could be a better name? `check_credential_only`?